### DISCUSSION TITLE
Modeling Agents and Personas/Pseudonyms

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#340] [General]
# Documents
## [Pseudonyms and persons and WEM Clements](https://github.com/uwlib-cams/MARC2RDA/blob/main/ModelingReference/Pseudonyms%20and%20persons%20and%20WEM.Clements.pdf)
# Correspondence and Notes
## Initial question from Sofia Zapounidou:
Since you are working on a project regarding RDA representation of bibliographic data I would like your opinion on the representation of pseudonyms. <br>

The aim is to be able to model a Person with all his/her pseudonyms, and also which version of his/her name is used in a specific Work or Expression. This mechanism already exists for Manifestation, but I have not found anything for Work and Expression entities. In case you know that there is way for representing the case I am describing, please tell me so. <br>

With the implementation of this scenario, (I hope that) it will be possible to cluster all Works authored by a Person and provide also sub-clusters under Pseudonyms or variations of the name. Variations of names are important for Greek Name Authorities especially for priests and monks that change their names and titles frequently, e.g. 'Monk A of monastery name' during 1920-1930, 'Bishop A of Thessaloniki' during 1931-1952, and 'ArchBishop A of Greece' during 1953-1960. <br>

Please check the attached image and start reading from the Manifestation level (bottom-up). I have included with dashed arrows what I would expect from RDA and I did not find. <br>
## Response from Crystal
Your question is an interesting one, and we discussed it at our MARC21-to-LRM/RDA/RDF mapping meeting this morning. <br>

We think pseudonyms are recorded in RDA as nomen strings for separate Nomens, related to one common entity for a Person, and the Person entity is related to the Works, Manifestations, and Expressions associated with that person and all of their alternate identities/pseudonyms. While pseudonyms can be represented as structured strings in authorized access points for Works/Expressions, and included in values of other properties such as statements of responsibility for Manifestations, a direct relationship between a given nomen string for a Person and that Person’s associated Works/Expressions does not exist in RDA. There are possibilities for relating nomens for different entities to one another. I’ve included a PDF illustration as an attachment, “Pseudonyms and persons and WEM”. The properties I include are just for illustrative purposes, as the exact properties used depend on implementation/application profiles. <br>

Sita Bhagwandin (National Library of the Netherlands) shared a powerpoint with us during the discussion that speaks to the relationships between a person and their personas/identities. The document (attached), “Person and personas or identities” was authored by Gordon Dunsire, and I’ve cc’d him on this email as well. Our mapping work does not cover authority data, so perhaps Gordon has more to say on the subject or can provide a more accurate explanation.<br>

**Comment by GordonDunsire** [comment]
See also the PCC/Library of Congress training presentation "Bibliographic Identities (Personas)", available for download via Google (I can't find a direct link).

It discusses one method of modelling a "persona", as a nomen cluster that is associated with a distinct context of use. The context of use is the clustering device, accommodated by [Nomen: context of use](http://rdaregistry.info/Elements/n/P80067).

An additional clustering device is implied in my slides: the RDA distinction of three categories of appellation/nomen as unstructured name/title, structured access point, and (string) identifier.

For example, "BBC", "B.B.C.", and "British Broadcasting Corporation" form a sub-cluster of names used in manifestations by a specific collective agent.

As noted, VIAF forms sub-clusters of access points. In theory, there should be no requirement for an identifier sub-cluster, but it may arise in practice. For example, the 'hyphenated' and 'spaced' variations in ISBN may be considered as valid identifiers from the same VES.

A nomen cluster is not treated as real-world object (entity) in LRM/RDA, so it cannot be the object of a metadata statement that uses a relationship element. This includes elements with range Nomen: a nomen cluster is not a nomen. It can only be associated with an attribute (datatype) element that has a literal string value.

The domain (subject) of the association is Work. The range is a string that represents the nomen cluster.

There are two methods of stating such an association.

1. Application of distinct string encoding schemes for values of [Work: authorized access point for work](http://rdaregistry.info/Elements/w/P10331). The distinction is made in the choice of value of [Person: authorized access point for person](http://rdaregistry.info/Elements/a/P50411), which depends on the value(s) of [Person: name of person](http://rdaregistry.info/Elements/a/P50111) used in manifestations that embody the work.

Work1 has preferred title of work NomenW11 .
NomenW11 has nomen string "Harry Potter and the philosopher’s stone" .
Work1 has author person PersonA .
PersonA has authorized access point for person NomenPA1 .
NomenPA1 has nomen string "Rowling, J. K." .
Work1 has authorized access point for work NomenW12 .
NomenW12 has nomen string "Rowling, J.K. Harry Potter and the philosopher's stone" .

Work2 has preferred title of work NomenW21 .
NomenW21 has nomen string "The cuckoo's call" .
Work2 has author person PersonA .
PersonA has authorized access point for person NomenPA2 .
NomenPA2 has nomen string "Galbraith, Robert" .
Work2 has authorized access point for work NomenW22 .
NomenW22 has nomen string "Galbraith, Robert. The cuckoo's call" .

The 'persona' nomen is effectively embedded in a work nomen.

This method requires guidance and instructions on the differentiation and application of distinct string encoding schemes for the same element.

2. Use an appellation of work group.

Work1 has authorized access point for work group NomenWG1 .
NomenWG1 has nomen string "Rowling, J.K." .
Work2 has authorized access point for work group NomenWG2 .
NomenWG2 has nomen string "Galbraith, Robert" .

These are distinct from other authorized access points specific to each work.

The 'persona' nomen is effectively an added work nomen.

This method requires guidance and instructions on the differentiation and application of distinct nomens for the same entity.

It also requires separate maintenance from direct nomens, and is the equivalent of maintaining a vocabulary encoding scheme for nomen clusters.

All of the nomens are ultimately 'just' strings; the strings are manipulated and concatenated to form other strings using string encoding schemes. Which strings, what manipulations, and the collocation utilities they support are all determined by the naming conventions of local cultures.

This includes the expectations of end-users, creators of works and manifestations, and cataloguers.

I have also discussed aspects of this in [Reconstructing authorities: new approaches to the management and use of authority data](https://morepress.unizd.hr/books/press/catalog/view/62/61/985-1)

**Comment by renevoorburg** [comment]
In our linked data based approach to RDA we aim not to rely on approaches based on access points.

We are examining an approach to nomen clusters that relates all nomens in a 'persona' nomen cluster using "equivalent to". With that approach, each nomen cluster has a central nomen. The URI of the central nomen provides, in an indirect way, a reference to the persona. Idea is that, in our application profile, for example "Rowling" and "Galbraith" are not linked using "equivalent to" but  in another example, "BBC", "B.B.C.", and "British Broadcasting Corporation" are linked with "equivalent to".

See the attached pdf for an example.

[cluster.pdf](https://github.com/uwlib-cams/MARC2RDA/files/11439828/cluster.pdf)

**Reply by GordonDunsire** [reply]
This approach can be simplified further by avoiding minting nomens except for the "central nomens". In many applications there is no requirement to say anything about the variant nomen strings that are clustered, so there is no need to mint a nomen for the variant. Applications like VIAF that want to record the language and script of variant nomen strings can use XML language tags.

The attached png is the same example, simplified.

![personas](https://github.com/uwlib-cams/MARC2RDA/assets/1566119/3891f258-6fb2-451d-8c7f-9a3eb63513c6)

**Comment by renevoorburg** [comment]
Thank you Gordon. That is an approach I was also thinking of, providing IRI's only to the central Nomen (that was what the drawing of the 'label' icon in the PDF was signifying - not really self explanatory, sorry for that  :-) and using blank nodes for the others. 

It is an approach that will work for us I guess, but even when the 'persona' isn't a concept in RDA, I wished we could also come up with an approach that enables us to  'extract' persona's from the RDA graph. Omitting the "equivalent to" relations between the central nomens helps to extract a persona with  "all nomens linking to central nomen(that refers to an rda:Person) form the cluster of names of a Persona". Adding "equivalent to" relations between the central nomens too would confuse that approach. 

Also, a weakness of relying on only the central nomens having a URI is that it breaks when in the future, for one reason or the other, - using blank nodes can have downsides - we might want to give all nomes a URI.

**Reply by GordonDunsire** [reply]
You should be able to isolate the "nomen/persona" nodes by filtering on "has real identity of person" and "has alternate identity of person".

You can search for personas by the nomen strings of the "is equivalent to" cluster (and the "has nomen string" of the persona, implied by the node caption; in this case an authorized access point).

The "is equivalent to" links between persona nodes are not necessary, but provide a shortcut cross-reference between personas (e.g. see also). If they cannot be filtered programatically when searching for personas, they should be excluded from the graph. Note that a filter is trivial if the datatype and object versions of "is equivalent to" are used.

An isolated persona node is effectively a "name/nomen authority" for the rwo person, and functions as a work clustering device (to keep Victoria amused). If the broad "has related work of nomen" is considered to be too vulnerable to ad hoc occurrences, then a refinement such as "has related work of persona" element can be introduced (although that suggestion is vulnerable to bad eyesight :-).

This approach can be extended to reified clustered nomen strings by refining the "is equivalent to" element to, say "is equivalent persona to" - "relates a nomen to a nomen that is an appellation of the same persona as another nomen". This refined element is used to cluster around a persona, and the "is equivalent to" element links the personas.

![personasPlus](https://github.com/uwlib-cams/MARC2RDA/assets/1566119/a2cdb907-5969-41b6-8aa1-d0cf0ab41fd7)

However, reifying all of the clustered nomen strings will slow the extraction of persona indexes, etc. I don't think the risk of losing the de-referenced persona graph is high enough to warrant the expense, because the personas are already established, abeit very messily related, in large scale systems such as VIAF.

Note that this introduces the concept of "persona" to RDA, as a kind or category of Nomen. Applications can use any local categories they want, and formalize them as value vocabularies. RDA Toolkit itself uses "kinds" of entity, such as "aggregating work". In the case of Nomen, there are no current "kinds" because the harder semantic approach was taken to refine the "has appellation of RDA entity" element into the library-oriented name/title, access point, and identifier strings. "[A deeper dive into Nomen and appellations](http://www.rda-rsc.org/sites/all/files/Dunsire%20Deeper%20Dive%20into%20Nomens%2028%20Jan.pdf)" has more info.

This all looks appropriate for the Toolkit Community Resources area :-)

**Reply by renevoorburg** [reply]
Thank you Gordon. Indeed, "has real identity of person" and "has alternate identity of person" relations can be used to find the (central Nomen node of) a persona. I think it will be a good idea to use that approach for the application profile we are working on. And indeed, it is a better solution than using Nomen clusters defined by "equivalent to" - relations, as I suggested earlier.

I think we should also define what a persona is, in our application profile. Without delving into definitions, I am inclined to say to that an Agent (specifically a Person) can have multiple identities. Each identity of a person is a / or is 'reflected by' a persona. A persona is closely related to, but not equal to a name (Nomen). For example, 'Multatulli' and 'Muraturi' are separate Nomens, but are part of the same 'Multatuli' persona. So a persona is reflected (no strictly identical to) to a cluster of Nomens. A persona might have just one Nomen. Some personas provide virtual birthdates and biographies, other personas 'are' just a single Nomen(string), a pseudonym. A persona is a persona, regardless whether it relates to the 'real' or 'alternative' identity, but of course in the bibliographical domain personas become a thing to distinguish or to talk about when there exists more than one for a Person.

Personas might be seen as a 'meta class' based on RDA: RDA doesn't have a class that is identical to what we see as a persona but specific queries can be defined in RDA (or at least in our application profile) that will provide the personas for an Agent.

From a very practical perspective; personas help us, indeed, with the Victoria use case where one is interested in works by a specific persona, iso a specific Agent. The persona approach matches closely with how our current name authority records are organized and with approaches like VIAF, so it helps us to integrate with that.

**Reply by renevoorburg** [reply]
An example of this Persona-approach in RDF -work in progress- can be found here:  https://github.com/renevoorburg/rdanl/blob/master/rdf/examples/personae_rdanl.ttl

**Reply by GordonDunsire** [reply]
@renevoorburg:  It is a pity that this uses blank nodes :-), and I'm not sure that they are necessary in this context.

The only statement that is made about the nomens is the language, and this can be accommodated using a language tag:

`rdanl:equivalentTo "Multatuli" ,
 [
 a rdanl:Nomen ;
 rdanl:nomenString "穆尔塔图里";
 rdanl:languageOfNomen "zh-CN"
 ]
 ,
 [
 a rdanl:Nomen ;
 rdanl:nomenString "Mu'ertatuli";
 rdanl:languageOfNomen "zh-Latn-pinyin"
 ] ;`
	
=> `rdanl:equivalentTo "Multatuli"@zh-CN , "Mu'ertatuli"@zh-Latn-pinyin ;`	

I cannot find "zh-CN" in the [IANA sub tag registry](http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry). Are these "old" tags?

Strictly-speaking, "zh-Latn-pinyin" is a script, so the blank node would better state:

`rdanl:scriptOfNomen "zh-Latn-pinyin" ;`

There is an interesting discussion about the language of nomens on the CIDOC CRM SIG listserv:

[error in RDFS for 7.1.1 for the class that is a subclass of E41 and E33](https://www.mail-archive.com/crm-sig@ics.forth.gr/msg04754.html)
followed by [Issue 624, linguistic Appellation](https://www.mail-archive.com/crm-sig@ics.forth.gr/msg04859.html)

**Reply by renevoorburg** [reply]
Thank you @GordonDunsire ! I had a mental note that I needed to check those language codes  and the "zh-Latn-pinyin" thing. They are probably not correct indeed. 
One of the ideas behind this very example was to demonstrate that the equivalent Nomens can be recorded as Literals but also als entities (for the sake of simplicity - in a certain sense - as blank nodes). Indeed, there is no need for that approach when using the RDF language tag, which would be a more RDF minded solution here. 

However, adding a `rdanl:scriptOfNomen "zh-Latn-pinyin" ; ` statement revives the need for Nomen entities :-)
